lms/templates/seq_module.html

```diff
@@ -44,10 +44,11 @@
      ${item['content'] | h}
   </div>
   % endfor
-  <div id="seq_content" role="tabpanel"></div>
+  <div id="sr-is-focusable" tabindex="-1"></div>
```

Comment:
@talbs I'd considered this. However @cptvitamin said that having something `.sr`d would hide it and negate the focusable qualities. I opted for an `id` in this case... but a class might work better - only not `.sr`. Thoughts from either of you on a proper name?